src/components/views/dialogs/NewSessionReviewDialog.js

```diff
@@ -33,7 +34,24 @@ export default class NewSessionReviewDialog extends React.PureComponent {
     }
 
     onCancelClick = () => {
-        this.props.onFinished(false);
+        const ErrorDialog = sdk.getComponent("dialogs.ErrorDialog");
+        Modal.createTrackedDialog("Verification failed", "insecure", ErrorDialog, {
+            headerImage: require("../../../../res/img/e2e/warning.svg"),
+            title: _t("Your account is not secure"),
+            description: <div>
+                {_t("One of the following may be compromised:")}
+                <ul>
+                    <li>{_t("Your password")}</li>
+                    <li>{_t("Your homeserver")}</li>
+                    <li>{_t("This device, or the other device")}</li>
```

Comment:
We have since switched to using the term "session" instead of "device", so let's change that here. A lot of designs might have older wording, so it's a bit confusing.

```suggestion
                    <li>{_t("This session, or the other session")}</li>
```